src/main/java/org/ssu/edu/teachua/db/repository/impl/ClubsCategoryDAOImpl.java

```diff
@@ -0,0 +1,19 @@
+package org.ssu.edu.teachua.db.repository.impl;
+
+import org.ssu.edu.teachua.db.entities.Club;
+
+import java.util.Map;
+import java.util.TreeMap;
+
+public class ClubsCategoryDAOImpl extends AbstractDAOImpl<Club> {
+    private static final Map<String, AbstractDAOImpl> entitiesDaoImpl;
+
+    static {
+        entitiesDaoImpl = new TreeMap<>();
+        entitiesDaoImpl.put("categories", new CategoryDAOImpl());
+    }
```

Comment:
maybe we need to add here :
**entitiesDaoImpl.put("clubs", new ClubDAOImpl());**